qutebrowser/misc/sessions.py

```diff
@@ -416,6 +435,31 @@ def _load_tab(self, new_tab, data):
         except ValueError as e:
             raise SessionError(e)
 
+    def _load_tree(self, tabbed_browser, tree_data):
+        root_data = tree_data.get(0)   # root node always has uid 0
```

Comment:
Not necessarily. After trying to use this branch, i ended up with a bunch of windows one of which had:
```
- Group
- - Tab 1
- - Tab 2
- - Tab 3
- Group
- - Tab 1
- - Tab 2
- - Tab 3
```

... etc. 

Apparently, the session file had the root with uid 17.

What i did to fix this issue is, take the first key as such:
```
        tree_keys = list(tree_data.keys())
        if not tree_keys:
            return

        root_data = tree_data.get(tree_keys[0])   # root node always has uid 0
```